extension/src/LanguageFunctions.ts

```diff
@@ -1239,7 +1240,7 @@ function getTransUnitLineType(textLine: string): TransUnitElementType {
 }
 
 function getTransUnitElementMaxLines(): number {
-  return 7; // Must be increased if we add new note types
+  return 6; // Must be increased if we add new note types. But custom notes are not considered
```

Comment:
With this change, this comment could be removed, right?

Reply:
> Just a few comments to address :)

Good suggestions :) I adressed them in the last commit https://github.com/jwikman/nab-al-tools/pull/217/commits/046671bf5f190fa4c886d25bfe545b0d435b01bf 
@theschitz your suggestions are adressed as well! I think that's it now :)